### ISSUE TITLE
setDataSource() doesn't take array as argument

### ISSUE THREAD

**Issue by vastlik** [ublaboo/datagrid#52]
Hi there,
i tried:
```php
public function createComponentSimpleGrid($name)
{
	$grid = new DataGrid($this, $name);
	$data = array("id"=> 1);
	$grid->setDataSource($data);
	$grid->addColumnText("id", "id");

}
```

but I am getting:
```
Argument 1 passed to Ublaboo\DataGrid\DataModel::__construct() must be an instance of Ublaboo\DataGrid\DataSource\IDataSource, instance of Ublaboo\DataGrid\DataSource\ArrayDataSource given
```

when i use this
```php
$data_arr = array(
			0 => array(
				'id' => 5,
				'name' => 'Peter'
			),
			1 => array(
				'id' => 5,
				'name' => 'John'
			),
			2 => array(
				'id' => 5,
				'name' => 'Vern'
			)
		);

		$source = new ArrayDataSource($data_arr); //limitation for array

		$grid = new DataGrid($this, $name);

		$grid->setDataSource($source);
		$grid->addColumnText("id", "id");
```
I am getting 
```
DataGrid can not take [Ublaboo\DataGrid\DataSource\ArrayDataSource] as data source. search► 
```

**Comment by paveljanda** [comment]
Yes, your example 1 is correct.

ArrayDataSource has to implement IDataSource interface. I will fix that in the evening. Or you can send a PR. ☺

**Comment by paveljanda** [comment]
Done.

**Comment by vastlik** [comment]
If you want, you can close issues via. commands
```
This closes #34, closes #23, and closes example_user/example_repo#42
```
so you don't have to close it manually.
[https://help.github.com/articles/closing-issues-via-commit-messages/](documentation)

**Comment by paveljanda** [comment]
Oh I completely forgot that. Thanks. :)
